src/services/rolls/constants/gqlTypeDefs.ts

```diff
@@ -41,6 +41,8 @@ const typeDefs = gql`
     reward: Int!
     rewardDenomination: String!
     createdAt: String # unix timestamp
+    startedAt: String # unix timestamp
```

Comment:
startDate needs to be added, but startedAt does provide relevant information as well obviously. so i'd include both?